docs/docs/guides/kubescape-for-teenagers/kubescape-for-teenagers.md

```
@@ -1,11 +1,11 @@
-#Kubescape for Teenagers
+# Kubescape for Teenagers
 
-Around the time of Kubecon + CloudNativeCon Paris 2024, friend of Kubescape [Antony KERVAZO-CANUT](https://github.com/AntonyCanut/AntonyCanut) wrote one of his guides for teenagers on Kubescape. 
+Around the time of Kubecon + CloudNativeCon Paris 2024, friend of Kubescape [Antony KERVAZO-CANUT](https://github.com/AntonyCanut/AntonyCanut) wrote one of his guides for teenagers on Kubescape.
 
 To get an idea of the basics of Kubescape, feel free to download the guide below. You can choose between the French and English versions.
 
 Please note that Kubescape is changing quickly and adding functionality all the time. For more details, visit our [documentation](https://kubescape.io/docs/) and our [Repo on Github](https://github.com/kubescape).
 
-You can download the English version [here](/guides/kubescape-for-teenagers/kubescape-for-teenagers-en.pdf)
+You can download the English version [here](Kubescape%20for%20teenagers%20EN.pdf).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)